mne/time_frequency/tfr.py

```diff
@@ -605,36 +960,12 @@ def __init__(self, info, data, times, freqs, nave, comment=None,
             raise ValueError("Number of times and data size don't match"
                              " (%d != %d)." % (n_times, len(times)))
         self.data = data
-        self.times = np.asarray(times)
-        self.freqs = np.asarray(freqs)
+        self.times = np.array(times)
```

Comment:
OOC, why use `array` instead of `asarray`? Is there a preferred behavior to the former?